starter_project/mutation_handler.py

```diff
@@ -0,0 +1,175 @@
+        lenition_map: A string_map or union of transducers for lenition.
+        eclipsis_triggers: A union of strings of eclipsis triggers.
+        eclipsis_map: A string_map or union of transducers for eclipsis.
+        preprocessing: A union of language-specific transducers
+            for preprocessing a string.
+        postprocessing: A union of language-specific transducers
+            for postprocessing a string.
+        sigma_star: A cyclic, unweighted acceptor representing
+            the closure over the alphabet.
+    """
+
+    def __init__(self: str) -> None:
+        """Initialize attributes to none, other methods will set as needed."""
+        self.graphemes = "None"
+        self.soft_triggers = "None"
+        self.soft_map = "None"
+        self.hard_triggers = "None"
+        self.hard_map = "None"
+        self.spirant_triggers = "None"
+        self.spirant_map = "None"
+        self.mixed_triggers = "None"
+        self.mixed_map = "None"
+        self.nasal_triggers = "None"
+        self.nasal_map = "None"
+        self.lenition_triggers = "None"
+        self.lenition_map = "None"
+        self.eclipsis_triggers = "None"
+        self.eclipsis_map = "None"
+        self.preprocessing = "None"
+        self.postprocessing = "None"
+        self.sigma_star = union(*("[{}]".format(i) for i in range(1, 256))
+                                ).optimize().closure()
+
+
+    def breton_handler(self):
+        """Handler for Breton-specific information."""
+
+        self.graphemes = union("a", "b", "ch", "c'h", "d", "e", "f", "g",
+                               "h", "i", "j", "k", "l", "m", "n", "o", "p",
+                               "r", "s", "t", "u", "v", "w", "y", "z",
+                               "â", "ê", "î", "ô", "û", "ù", "ü", "ñ")
+
+        self.soft_triggers = union("da", "dre", "a", "war", "dindan",
+                                   "eme", "en ur", "ne", "na", "ra",
+                                   "en em", "daou", "div", "pa",
+                                   "pe", "an holl", "re",
+                                   #"e", "tra", "ez",
+                                  )
+        self.hard_triggers = union("ho", "az", "da'z",
+                                   #"ez",
+                                  )
+        self.spirant_triggers = union("he", "va", "tri", "teir", "pevar",
+                                      "peder", "nav", "hon",
+                                      #"ma", "o"
+                                     )
+        self.mixed_triggers = union("o", "e", "ma")
+
+        self.soft_map = string_map((
+            ("p", "b"),
+            ("t", "d"),
+            ("k", "g"),
+            ("gw", "w"),
+            ("m", "v"),
+            ))
+        self.hard_map = string_map((
+            ("b", "p"),
+            ("d", "t"),
+            ("g", "k"),
+            #("gw", "kw"), # remove because g->k exists?
```

Comment:
Move challenges/questions to README or other doc for follow up later.

Reply:
I added a short README file with some of the issues encountered for this implementation.